TTS/utils/generic_utils.py

```diff
@@ -169,7 +169,7 @@ def format_aux_input(def_args: Dict, kwargs: Dict) -> Dict:
     """
     for name in def_args:
```

Comment:
Maybe instead change the function definition to something like 
```python
    from copy import deepcopy
----
    kwargs = deepcopy(kwargs)
    for key in def_args:
        if key in kwargs:
            def_args[key] = kwargs[key] if kwargs[key] is not None else def_args[key]
            del kwargs[key]
    
    def_args.update(kwargs)
    return def_args
```

then you can leave the initial aux_input value in the inference parameter, and the function is called in the same order as your change.  `format_aux_input(default_input_dict, aux_input)`

Reply:
How about:

```python
kwargs = kwargs.copy()
for key in def_args:
    if key not in kwargs or kwargs[key] is None:
        kwargs[key] = def_args[key]
return kwargs
```

The description of the function is that it updates `kwargs` with missing values from def_args, and while your suggestion is dual, I think it's better for the code to line up with the description of the function. Your proposal version is "take the default args and override every key with the value from kwargs if it's there."

They're equivalent in terms of functionality and not changing things in-place, so I don't think it's super-important which way to choose. WDYT?

Reply:
Yeah! Works! It is only used at one or two places, so I don't think that is a major problem, but this also fixes a tiny bug it had! 
Thanks for the help :)